JDK Download Url Is 404 When Clicked In VsCode

Got this error message:

```
> cordova run android
[cordova]  Android Studio project detected
[cordova]  Failed to run "javac -version", make sure that you have a JDK installed.
[cordova]  You can get it from: http://www.oracle.com/technetwork/java/javase/downloads.
[cordova]
[ERROR] An error occurred while running subprocess cordova.

        cordova run android exited with exit code 1.

        Re-running this command with the --verbose flag may provide more information.
```

Ctrl-clicking the link in vs-code opens up the url with the `.` on the end and is a 404 error.

Not sure if this is a vscode bug, but including the dot seems unnecessary and likely to trip up clients.

Raising it here for you to decide if it's an issue or not.

Prompt fix! I'm still going through the process though and it seems that the url is wrong anyway.

After installing the JDK from [that url](http://www.oracle.com/technetwork/java/javase/downloads), I get this now:

```
> cordova run android
[cordova]  Android Studio project detected
[cordova]  ANDROID_HOME=C:\Users\username\AppData\Local\Android\sdk
[cordova]  JAVA_HOME=C:\Program Files\java\jdk-11.0.2
[cordova]  Requirements check failed for JDK 1.8
[ERROR] An error occurred while running subprocess cordova.
```

My suspicion is that it is a bug that is failing to parse a double-digit version number now that its on v11, however that doesn't matter as the docs are clear:

https://cordova.apache.org/docs/en/latest/guide/platforms/android/index.html#java-development-kit-jdk

They say that you have to install JDK v8. 

The v8 download links *are* further down on the [main download url](http://www.oracle.com/technetwork/java/javase/downloads) but it's not intuitive that you should install an old version when the top download is an lts release.

A possible replacement link, that only includes the v8 downloads is:

https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html

--- a/bin/templates/cordova/lib/check_reqs.js
+++ b/bin/templates/cordova/lib/check_reqs.js
@@ -204,9 +204,11 @@ module.exports.check_java = function () {
             }, () => {
                 var msg =
                 'Failed to run "javac -version", make sure that you have a JDK installed.\n' +
-                'You can get it from: http://www.oracle.com/technetwork/java/javase/downloads.\n';
+                'You can get it from the following location:\n' +
+                'https://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html';
                 if (process.env['JAVA_HOME']) {
-                    msg += 'Your JAVA_HOME is invalid: ' + process.env['JAVA_HOME'] + '\n';
+                    msg += '\n\n';
+                    msg += 'Your JAVA_HOME is invalid: ' + process.env['JAVA_HOME'];
                 }
                 throw new CordovaError(msg);
             });

